firestore/google/cloud/firestore_v1beta1/client.py

```diff
@@ -252,10 +251,8 @@ def write_option(**kwargs):
         :meth:`~.DocumentReference.update` and
         :meth:`~.DocumentReference.delete`.
 
-        Exactly one of three keyword arguments must be provided:
+        One of the following two keyword arguments must be provided:
```

Comment:
Can you just remove `two` here? (`three` should've been removed from the previous description)

Reasoning: the count is clear from the bullets but is just one more thing we have to "maintain"